public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ConfigEditor.tsx

```diff
@@ -127,6 +128,8 @@ export class ConfigEditor extends PureComponent<Props, State> {
     // This is bad, causes so many messy typing issues everwhere..
     options.secureJsonData = (options.secureJsonData || {}) as AzureDataSourceSecureJsonData;
 
+    const isInsightsConfigured = isAppInsightsConfigured(options);
```

Comment:
Maybe add `useState` to not have the section disappear immediately as as user clears input?

Reply:
Good shout. I did this for Query Editor but not here 😅